transforms/computed_link_description.py

```diff
@@ -0,0 +1,25 @@
+from infrahub_sdk.transforms import InfrahubTransform
+
+
+class ComputedLinkDescription(InfrahubTransform):
+    query = "computed_link_description"
+    url = "computed_link_description"
+
+    async def transform(self, data):
+        interface_dict: dict = data["InfraInterface"]["edges"][0]["node"]
+
+        description = f"{interface_dict['name']['value']}.{interface_dict['device']['node']['name']['value']}"
+        # Build the corresponding connected_endpoint if it exists.
+        if interface_dict.get("connected_endpoint"):
+            connected_endpoint = await self.client.get(
+                kind=interface_dict["connected_endpoint"]["node"]["__typename"],
+                id=interface_dict["connected_endpoint"]["node"]["id"],
+            )
+            await connected_endpoint.device.fetch()
```

Comment:
I get the following error if I don't call the `.fetch` method.

```python
(Pdb) connected_endpoint.device.peer
*** infrahub_sdk.exceptions.NodeNotFoundError: 
        Unable to find the node in the Store
        None | InfraDevice | {'key': ['181b9049-cf6c-7947-3be3-c518a988c5bd']}
```

Reading the documentation, that should be available due to it being a cardinality of one relationship between `InfraInterfaceL3` and `InfraDevice`, but I raised this comment to understand better.